Jenkinsfile

```diff
@@ -1,3 +1,43 @@
+def get_build_info() {
+  pr_branch = ''
+  if (env.CHANGE_BRANCH != null) {
+    pr_branch = " (${env.CHANGE_BRANCH})"
+  }
+  build_info = "#${env.BUILD_NUMBER} of <${env.BUILD_URL}|${env.JOB_NAME}>${pr_branch}"
+  return build_info
+}
+
+def slack_send(color, message) {
+  /* Slack channel names are limited to 21 characters */
+  CHANNEL_MAX_LEN = 21
+
+  channel = "${env.JOB_NAME}".tokenize('/')[0]
+  channel = channel.replace('lisk-', 'lisk-ci-')
+  if ( channel.size() > CHANNEL_MAX_LEN ) {
+     channel = channel.substring(0, CHANNEL_MAX_LEN)
+  }
+
+  echo "[slack_send] channel: ${channel} "
+  slackSend color: "${color}", message: "${message}", channel: "${channel}"
+}
+
+def recovery() {
+  if (currentBuild.result == null || currentBuild.result == 'SUCCESS') {
```

Comment:
Do we want notifications only on recovery? This was a requirements for some other project, not sure about lisk-explorer though.

Reply:
This is a good question

Reply:
@MichalTuleja How do we want this to behave?